src/components/ModalForm.tsx

```diff
@@ -0,0 +1,20 @@
+import React, {useState, FormEvent, Component} from 'react'
+
+interface Props {
+    onSubmitUsername: (e: FormEvent, username: string) => void
+}
+
+export default function ModalContent(props: Props) {
+    const [username, setUsername] = useState('')
+    return (
+        <div className="modalContent">
+            <form onSubmit={e => props.onSubmitUsername(e, username)}>
+                <label style={{color: 'white'}}>
```

Comment:
because inline styling is near impossible to override and it clutters the code, its a common practice to avoid them, and only use them as last resort solutions. can you create a css file for the component like you did for the `Modal` component and style there?